modules/oc-mirror-mirror-to-mirror.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-mirror-to-mirror_{context}"]
+= Mirroring from mirror to mirror
+
+You can use the oc-mirror plug-in to mirror an image set directly to a target mirror registry that is accessible during image set creation.
+
+// TODO: This note might be obsolete come GA, if they change the impl
+[IMPORTANT]
+====
+Depending on the configuration specified in the image set configuration file, using oc-mirror to mirror images might download several hundreds of gigabytes of data to disk before mirroring to the destination mirror registry.
+
+The initial image set to populate the mirror registry is often the largest. Subsequent executions of oc-mirror only download the changes since the previous execution, so the generated image set is often a smaller size.
+====
+
+You are required to specify a storage backend in the image set configuration file. This storage backend can be a local directory or a Docker v2 registry. The oc-mirror plug-in stores metadata in this storage backend during image set creation.
+
+[IMPORTANT]
+====
+Do not delete or modify the metadata that is generated by the oc-mirror plug-in executions. You must use the same storage backend on subsequent executions of oc-mirror to the same mirror registry.
+====
+
+.Prerequisites
+
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plug-in.
+* You have created the image set configuration file.
+
+* Run the `oc mirror` command to mirror the images from the specified image set configuration to a specified registry:
++
+[source,terminal]
+----
+$ oc mirror --config=./imageset-config.yaml \ <1>
+  docker://registry.example:5000/             <2>
```

Comment:
some here , the docker url should be : "docker://registry.example:5000"  or "docker://registry.example:5000/namespace" , if no namespace should remove '/'

Reply:
Updated!